src/Surfnet/StepupSelfService/SelfServiceBundle/Service/RemoteVetting/IdentityProviderFactory.php

```diff
@@ -66,4 +78,18 @@ public function create($name)
 
         throw new InvalidRemoteVettingIdentityProviderException(sprintf("Invalid IdP requested '%s'", $name));
     }
+
+
+    /**
+     * @param string $name
+     * @return array
+     */
+    public function getAttributes($name)
```

Comment:
I strongly suggest you rename this to getAttributeMapping or something along those lines. This is a misleading name.

Reply:
Good suggestion, changed!